### ISSUE TITLE
Good doggos can fail to display due to file size

### ISSUE THREAD

**Issue by BenTheElder** [kubernetes/test-infra#6819]
See https://github.com/kubernetes/test-infra/pull/6814#issuecomment-365651499

Internet Cats in the `cat` plugin probably also have this issue.

/kind bug
/P 0

**Comment by stevekuznetsov** [comment]
/woof
 :(

**Comment by matthyx** [comment]
Maximum size for images is 10MB (bad gif in #6814 was 12MB).
https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/

**Comment by matthyx** [comment]
Yes, I have a PR coming... just not sure we need a unit test for it.
